backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/textual/SysMLElementSerializer.java

```diff
@@ -210,6 +211,25 @@ public String caseAnalysisCaseUsage(AnalysisCaseUsage analysisCaseUsage) {
         return "";
     }
 
+    @Override
+    public String caseTextualRepresentation(TextualRepresentation textualRepresentation) {
```

Comment:
Please add every new casXXX in every Switch class at the right location (they are all ordered alphabetically).
Please keep in mind for your next contributions.